package.json

```diff
@@ -1,5 +1,16 @@
 {
     "scripts": {
-      "test": ""
+        "build": "tsc",
+        "test": "npm run build && node ./out/test/runTest.js"
+    },
+    "devDependencies": {
+        "@types/glob": "^8.1.0",
+        "@types/mocha": "^10.0.9",
+        "@types/node": "^22.9.0",
+        "@types/vscode": "^1.95.0",
+        "@vscode/test-electron": "^2.4.1",
```

Comment:
What does glob, mocha do?

Reply:
It is the testing package for vscode extension. Will be needed in the future. should I delete them for now?

Reply:
Then you can keep it if its required